utils/build-script

```diff
@@ -470,6 +470,10 @@ class BuildScriptInvocation(object):
             impl_args += [
                 "--cross-compile-hosts", " ".join(args.cross_compile_hosts)]
 
+        if args.test_paths:
+            impl_args += [
+                "--test-paths", " ".join(args.test_paths)]
```

Comment:
Does this need the extra line-break here?

Reply:
It most likely doesn't, I just made it all look the same...